dlang.org.ddoc

```diff
@@ -214,6 +214,7 @@ MULTICOLS=$+
 MULTIROW_HEADER=<th rowspan="$1">$+</th>
 MREF=<a href="$(PHOBOS_PATH)$1$(UNDERSCORE_PREFIXED $+).html">$(D $1$(DOT_PREFIXED $+))</a>
 MREF_ALTTEXT=<a href="$(PHOBOS_PATH)$2$(UNDERSCORE_PREFIXED_SKIP $+).html">$1</a>
+MREF1=<a href="$(PHOBOS_PATH)$1.html">$(D $1)</a>
```

Comment:
Urgh, not a huge friend of more Ddoc macros, but I see the Ddoc limitations (we should really push Ddox forward).